static/third/bootstrap-typeahead/typeahead.js

```diff
@@ -86,6 +86,7 @@
 
   var Typeahead = function (element, options) {
     this.$element = $(element)
+    this.$window = $(window)
```

Comment:
I'm not sure if this serves a purpose or this was done to match the file's coding style, but I think a standalone `window.on('resize',...)` call should be fine.

Reply:
Yeah this was done just to match the file's coding style.